### PR TITLE
Add honeysqs

### PR DESCRIPTION
**Pull request by tredman**
Built this to ingest RDS events at honeycomb. This seems like the right home for it. It's simple but complete enough to use.

### CODE REVIEW

**Comment by nathanleclaire** [comment]
Hm, I think I might like to convert the existing `logbucket` directory into a `input` directory, with `logbucket` and `sqs` as subdirectories of that (just to organize code so that it's clear what the various "input connectors" are).

**Comment by nathanleclaire** [comment]
I'm generally curious if @maplebed feels it's a good fit for the suite of tools / model in general though. Having multiple inputs, e.g., via CloudWatch, might not be a bad thing, but it also might be out of scope for honeyaws.

File: cmd/honeysqs/main.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+						"MessageId": *message.MessageId,
+					}).
+					Error("error extracting event type from message")
+				continue
+			}
+			event := libhoney.NewEvent()
+			event.Add(translatedEvent)
+			err = event.Send()
+			// if we sent the event to honeycomb, we can delete it from the queue
+			if err == nil {
```
Comment (maplebed) [inline cmd/honeysqs/main.go]: no error on send doesn't actually mean that it was successfully sent to honeycomb, only that it was successfully enqueued. To actually verify it was sent you need to read from the response queue.

Comment (maplebed) [reply cmd/honeysqs/main.go]: (to be clear, I'm not suggesting that you need to check the response queue - changing the comment to not imply that the event was sent is probably fine.)

**Comment by maplebed** [comment]
in general I suspect there's going to be an integrations reckoning sometime and it will sort out what should be where. I'm losing a strong sense of what tools should be in which repo.

File: cmd/honeysqs/main.go (hunk 1 above)
Comment (tredman) [reply cmd/honeysqs/main.go]: It wasn't too much effort to check the response queue, so I added that (I really like the metadata in the libhoney Responses!)

**Comment by tredman** [comment]
Yeah I can see how this one doesn't quite fit in with the current theme of logbucket tailers, but I do like the idea keeping the AWS stuff together unless there's a strong reason to separate them. If people are curious about our AWS integrations, having them in one spot is convenient.

**Comment by tredman** [comment]
Abandoning this as we have an SNS integration here: https://github.com/honeycombio/agentless-integrations-for-aws

**Pull request closed without merging**